addons/hr_recruitment/models/res_users.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models
+
+class ResUsers(models.Model):
+    _inherit = 'res.users'
+
+    def _create_recruitment_interviewers(self):
+        if not self:
+            return
+        interviewer_group = self.env.ref('hr_recruitment.group_hr_recruitment_interviewer')
+        recruitment_groups = self.env.ref('hr_recruitment.group_hr_recruitment_user') | self.env.ref('hr_recruitment.group_hr_recruitment_manager')
```

Comment:
A manager is always a user due to the implied_ids attribute. :)
```suggestion
        recruitment_groups = self.env.ref('hr_recruitment.group_hr_recruitment_user')
```